Install/install.sh

```diff
@@ -42,9 +42,11 @@ fi
 echo ""
 echo "Welcome to BrickPi3 Installer."
 
-sudo chmod 755 /home/pi/Dexter/BrickPi3/Firmware/openocd/install_openocd_compiled.sh
+sudo chmod +x /home/pi/Dexter/BrickPi3/Firmware/openocd/install_openocd_compiled.sh
```

Comment:
Just a note: It is not necessary to chmod +x the file when it's being explicitely called with bash on the following line